### PR TITLE
fix bug in format indicatorfield

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#24357

## Description
added schemas for indicatorfield

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed a bug in the **format** command where indicatorField field.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:2]: What was the bug we fixed? Was it any fix or we just added tests to validate indicator fields?

Comment (ShahafBenYakir) [inline CHANGELOG.md:2]: I think we can remove this because we dod not fix anything here, the fix was already mentioned in this file right?

**Comment by moishce** [comment]
The bug was because there was a missing indicatorfield file in schemas,
But it has already been added by @hod-alpert ,
So I just arranged the tests that would really test what was needed

File: CHANGELOG.md (hunk 1 above)
Comment (ShahafBenYakir) [reply CHANGELOG.md]: Fixed a bug in the **format** command where `indicatorField` wasn't part of the SDK schemas.

**Comment by ShahafBenYakir** [comment]
Change the changelog to this https://github.com/demisto/demisto-sdk/pull/427/files/15cca8f07b394a427349281f69f39a1cfd53c02c..7948da98f8b091d78a3a8b95c60dda2069245e70#r424309615

**Pull request merged**